sim/rogue/talents.go

```diff
@@ -510,3 +601,50 @@ func (rogue *Rogue) registerKillingSpreeCD() {
 	}
 	rogue.registerKillingSpreeSpell()
 }
+
+func (rogue *Rogue) registerHonorAmongThieves() {
+	// When anyone in your group critically hits with a damage or healing spell or ability, you have a [33%/66%/100%] cahnce to gain a combo point on your current target. This effect cannot occur more than once per second.
+	if rogue.Talents.HonorAmongThieves == 0 {
+		return
+	}
+
+	// FIXME: Implement a flow of combo points that simulates the party/raid crit procs
+	procChance := []float64{0, 0.33, 0.66, 1}[rogue.Talents.HonorAmongThieves]
+	comboMetrics := rogue.NewComboPointMetrics(core.ActionID{SpellID: 51701})
+	honorAmongThievesID := core.ActionID{SpellID: 51701}
+
+	rogue.HonorAmongThievesDot = core.NewDot(core.Dot{
```

Comment:
Just use an aura starting a periodic action (gaining a combo point) instead, this seems to work, and doesn't require an extra spell.

Roughly ~
```
rogue.RegisterAura(core.Aura{
	Label:    "Honor Among Thieves",
	Duration: core.NeverExpires,
	OnReset: func(aura *core.Aura, sim *core.Simulation) {
		aura.Activate(sim)
	},
	OnGain: func(aura *core.Aura, sim *core.Simulation) {
		core.StartPeriodicAction(sim, core.PeriodicActionOptions{
			Period: time.Second,
			OnAction: func(sim *core.Simulation) {
				rogue.AddComboPoints(sim, 1, comboMetrics)
			},
		})
	},
})
```

Reply:
Works well with what you posted! Appreciate the snippet.